vhost-device-scmi/src/scmi.rs

```diff
@@ -666,11 +731,71 @@ impl ScmiHandler {
                 }
                 _ => Response::from(ReturnStatus::NotSupported),
             },
+            MessageType::Notification => Response::from(ReturnStatus::NotSupported),
             MessageType::Unsupported => Response::from(ReturnStatus::NotSupported),
         };
         ScmiResponse::from(request.header, response)
     }
 
+    pub fn get_device_eventfd_list(&self) -> Vec<(RawFd, u16)> {
+        let mut list = vec![];
+        for (eventfd, (protocol_id, device_index)) in self.event_fds.map.lock().unwrap().iter() {
+            if let Some(devices) = self.devices.0.lock().unwrap().get_mut(protocol_id) {
+                if let Some(device) = devices.get_mut(*device_index) {
+                    list.push((device.get_notify_fd().unwrap(), *eventfd));
+                }
+            }
+        }
+        list
+    }
+
+    pub fn get_max_eventfd(&self) -> u16 {
+        self.event_fds.available_device_event - 1
+    }
+
+    /// According to device_event, find out the device which will do notification.
+    /// Then call its notify function to return a ScmiResponse.
+    /// Now only SENSOR PROTOCOL can do notification.
+    /// And it only support SENSOR_UPDATE meesage.
```

Comment:
"And it supports only SENSOR_UPDATE message."

Reply:
Plus it's good formatting style to use backticks for uppercase constant names:

```markdown
And it supports only the `SENSOR_UPDATE` message.
```

Reply:
All right